site/layouts/_default/single.html

```diff
@@ -12,8 +12,15 @@
   <body data-spy="scroll" data-target="#d-navbar" data-offset="40">
     {{ partial "header.html" }}
     <div class="container">
-      <div id="about" class="row d-about col-md-9 offset-md-3">
-        {{ partial "about.md" | markdownify }}
+      <div class="row">
+        <div id="about" class="d-about col-md-9 offset-md-3">
+          <div class="d-rss pl-3 float-right">
+            <a href="index.xml">
+              <img style="height: 1em" src="./img/icon-rss.svg">
```

Comment:
Any reason not to specify the height in the CSS?

Reply:
Yes, good question. Putting the height on the image tag ensures that the image will have a reasonable height, even if the CSS fails to load.

Reply:
Ah, especially while loading. Got it.